amplify/team-provider-info.json

```diff
@@ -26,4 +26,4 @@
       }
     }
   }
-}
+}
```

Comment:
I don't understand what's changing here :thinking:

Reply:
Looks like the newline at the end of the file was removed. In general, we try to avoid having no newline at the end of files.

Reply:
That's only for these automatically generated files though? Otherwise I'm pretty sure we add a newline.

Reply:
I think showed up because you removed the previous changes, then again, I've also seen this act up from time to time.

I believe this will be fixed once we manage to integrate prettier _(or at least I hope)_